public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -341,10 +341,10 @@ export const LdapDrawerComponent = ({
                     onChange={(v) => {
                       setValue(
                         `${serverConfig}.root_ca_cert_value`,
-                        v.filter(({ v }) => typeof v === 'string')?.map(({ v }) => v)
+                        v.filter((v) => typeof v.value === 'string').map(({ value }) => String(value))
```

Comment:
```suggestion
                        v.filter((v) => typeof v.value === 'string').map(({ value }) => value)
```
You're already checking that it's a string, so no conversion is necessary.

Reply:
Changed as requested